oat-sa/extension-tao-itemqti
# fix: rules for deleting SCORE and MAXSCORE outcomes

Ticket: https://oat-sa.atlassian.net/browse/AUT-3693

## What's Changed

- Fixed rules for Deleting `SCORE` and `MAX_SCORE` and applying Default `SCORE` Value
- The `SCORE` and `MAX_SCORE` will be deleted when response processing is none and all other outcomes marked with `External scored` = `none`
-  The `SCORE` and `MAX_SCORE` will NOT be deleted when response processing is none and at least one outcome marked with `External scored` with `Human` or `External machine`
- The default value `0` will be applied for `SCORE` outcome for the case described just above

## How to test
- Create a test and check the cases described in `What's Changed` section.

Related PR in package: https://github.com/oat-sa/tao-item-runner-qti-fe/pull/405
**Deployed** to http://test-viktar.playground.kitchen.it.taocloud.org:41531/tao/Main/login
